### PR TITLE
Adding new problem

### PR DESCRIPTION
**Pull request by Jutulli**
For assignment 3 of SF-1

Woohoo !

### CODE REVIEW

File: count-repeat/meta.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "title": "CountRepeat",
+    "name": "countRepeat",
+    "difficulty": "Hard",
+    "author": "Jutulli",
+    "category": "lists"
+  }
```
Comment (Tariq-Menla) [inline count-repeat/meta.json:7]: the squiggly bracket at the end, make sure its in the same indentation level as the first one.

File: count-repeat/starter.py
Comment (Tariq-Menla) [inline count-repeat/starter.py:1]: if the pass thing is fine then this is good

**Review by Tariq-Menla — COMMENTED** [review]
make sure you fix the squiggly bracket at the end, make sure its in the same indentation level as the first one. (meta.json)

**Review by Tariq-Menla — COMMENTED** [review]
Should be good now! all looks great!

File: count-repeat/io.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "input": [" ", "x"],
+      "output": [0]
+    },
+    {
+      "input": ["hello", "l"],
+      "output": [1]
+    },
+    {
+        "input": ["RAHHahh", "h"],
+        "output": [1]
```
Comment (emaj9) [inline count-repeat/io.json:24]: outputs don't need a []

File: count-repeat/meta.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "title": "CountRepeat",
+    "name": "countRepeat",
+    "difficulty": "Hard",
+    "author": "Jutulli",
+    "category": "lists"
```
Comment (emaj9) [inline count-repeat/meta.json:6]: should be in the category "string-2"
difficulty should be "medium"
function names should follow python snake case, so here it should be name: count_repeat (with quotes). you need to change the starter.py as well

Comment (Jutulli) [reply count-repeat/meta.json]: Whoops, everything should be fixed now

**Pull request merged**